OSPSuite.Presentation.Importer/Core/DataFormat/DataFormat_TMetaData_C.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Text.RegularExpressions;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+using OSPSuite.Core.Importer;
+
+namespace OSPSuite.Presentation.Importer.Core.DataFormat
+{
+   public class DataFormat_TMetaData_C : AbstractColumnsDataFormat
```

Comment:
What is this name? LEt's be explicit. I could not say what format this is from the class name

Reply:
Comes from here [https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/639] . not sure how to possibly rename it to make more sense

Reply:
ok. I asked Juri to clarify. I do not know what this means